controller/search.go

```diff
@@ -33,14 +34,18 @@ func NewSearchController(service *goa.Service, db application.DB, configuration
 // Users runs the user search action.
 func (c *SearchController) Users(ctx *app.UsersSearchContext) error {
 
+	_, err := login.ContextIdentity(ctx)
+	if err != nil {
+		return jsonapi.JSONErrorResponse(ctx, errors.NewUnauthorizedError(err.Error()))
```

Comment:
We should add jwt security to design instead.

Reply:
Or you actually added it. Good. But then you don't have to check the token here.

Reply:
Yeah, but this makes sure that the IDE-integration token doesn't work here as well.

Reply:
And worth adding tests for deprovisioned users too.

Reply:
We could check the approved claim though.. But rejecting deprovisioned users is better IMO.

Reply:
Done d3e8660a2bbbb6c9349ce4d3ac7c78c4ff2d13a4
Yeah `approved` claim could have been checked, and might have been less expensive.
but, better be a bit expensive than leaky.